packages/babel-preset-expo/README.md

```diff
@@ -2,3 +2,22 @@
 
 If you have problems with the code in this repository, please file issues & bug reports
 at https://github.com/expo/expo. Thanks!
+
+## Options
+
+### `web.allowCommonJS`
+
+Enabling this will enable your project to run with older javascript syntax (ie: `module.exports`). By enabling this you will also break tree shaking and your bundle will be much larger than it could be. When the older syntax is in
```

Comment:
This might be clearer:
```suggestion
Enabling this option will allow your project to run with older JavaScript syntax (i.e. `module.exports`). This option will break tree shaking and increase your bundle size, but will eliminate the following error when `module.exports` is used:
```